client/src/app/site/motions/modules/motion-detail/components/motion-meta-data/motion-meta-data.component.ts

```diff
@@ -0,0 +1,308 @@
+    }
+
+    /**
+     * Sets the recommendation
+     *
+     * @param id Motion recommendation id
+     */
+    public setRecommendation(id: number): void {
+        this.repo.setRecommendation(this.motion, id);
+    }
+
+    public resetRecommendation(): void {
+        this.repo.resetRecommendation(this.motion);
+    }
+
+    /**
+     * triggers the update this motion's recommendation extension according to the current string
+     * in {@link newRecommendationExtension}
+     */
+    public setRecommendationExtension(nextExtension: string): void {
+        this.repo.setRecommendationExtension(this.motion, nextExtension);
+    }
+
+    /**
+     * Sets the category for current motion
+     *
+     * @param id Motion category id
+     */
+    public setCategory(id: number): void {
+        if (id === this.motion.category_id) {
+            id = null;
+        }
+        this.repo.setCategory(this.motion, id);
+    }
+
+    /**
+     * Adds or removes a tag to the current motion
+     *
+     * @param {MouseEvent} event
+     * @param {number} id Motion tag id
+     */
+    public setTag(event: MouseEvent, id: number): void {
+        event.stopPropagation();
+        this.repo.toggleTag(this.motion, id);
+    }
+
+    /**
+     * Add the current motion to a motion block
+     *
+     * @param id Motion block id
+     */
+    public setBlock(id: number): void {
+        if (id === this.motion.block_id) {
+            id = null;
+        }
+        this.repo.setBlock(this.motion, id);
+    }
+
+    /**
+     * Supports the motion (as requested user)
+     */
+    public support(): void {
+        this.repo.support(this.motion).catch(this.raiseError);
+    }
+
+    /**
+     * Unsupports the motion
+     */
+    public unsupport(): void {
+        this.repo.unsupport(this.motion).catch(this.raiseError);
+    }
+
+    /**
+     * Opens the dialog with all supporters.
+     * TODO: open dialog here!
+     */
+    public openSupportersDialog(): void {
+        this.showSupporters = !this.showSupporters;
+    }
+
+    /**
+     * Observes the repository for changes in the motion recommender
+     */
+    public setupRecommender(): void {
+        if (this.motion) {
+            const configKey: keyof Settings = this.motion.isStatuteAmendment()
+                ? 'motions_statute_recommendations_by'
+                : 'motions_recommendations_by';
+            if (this.recommenderSubscription) {
+                this.recommenderSubscription.unsubscribe();
+            }
+            this.recommenderSubscription = this.meetingSettingsService.get(configKey).subscribe(recommender => {
+                this.recommender = recommender;
+            });
+        }
+    }
+
+    /**
+     * Check if a recommendation can be followed. Checks for permissions and additionally if a recommentadion is present
+     */
+    public canFollowRecommendation(): boolean {
+        if (
+            this.perms.isAllowed('createpoll', this.motion) &&
```

Comment:
return this.perms.isAllowed('createpoll', this.motion) &&
this.motion.recommendation &&
this.motion.recommendation.recommendation_label

Reply:
Yeah, this makes sense. I don't know, who has written that.